src/applications/vaos/tests/e2e/community-care.cypress.spec.js

```diff
@@ -159,6 +160,55 @@ describe('Create new community care appointment', () => {
     // Click continue button
     cy.get('.usa-button').click();
 
+    // Check form requestBody is as expected
+    cy.wait('@appointmentRequests').should(xhr => {
+      expect(xhr.status).to.eq(200);
+      expect(xhr.url, 'post url').to.contain(
+        '/vaos/v0/appointment_requests?type=cc',
+      );
+
+      const request = xhr.requestBody;
+      expect(request)
+        .to.have.property('optionDate1')
+        .to.equal(
+          moment()
+            .add(5, 'days')
```

Comment:
I think we might get tripped up on weekends here. If 5 days from now is a Saturday, it won't be shown on the calendar and the following Monday will be chosen.

Reply:
Interesting problem. I ripped that code from the component code itself, https://github.com/department-of-veterans-affairs/vets-website/blob/dd14d78c992070dea49488c14407bba2cc67e99c/src/applications/vaos/new-appointment/components/DateTimeRequestPage.jsx#L130. Any ideas on what to do instead?

Reply:
It's a combination of that `minDate` prop plus the fact that our calendar doesn't display weekend days. This test picks the first available date, which most of the time will be the min date, but will sometimes be a day or two after.

I think you could account for that in the date you use. You can check the day of the week after you add 5 days and set it to the follow Monday if it's a weekend (https://momentjs.com/docs/#/get-set/day/)